### ISSUE TITLE
Jura dependency needs to be updated to use @accordproject/ergo

### ISSUE THREAD

**Issue by mttrbrts** [accordproject/cicero#48]
When using `cicero execute` with the latest 0.2.48 release of @accordproject/cicero-cli the following error occurs.

Rather than fixing the dependency to Jura, the cicero-engine library should use the latest version of the Ergo compiler.

```
09:24:17 - error:  Error: ENOENT: no such file or directory, open '/Users/matt/.nvm/versions/node/v8.11.1/lib/node_modules/@accordproject/cicero-cli/node_modules/node_modules/jura-engine/lib/juraruntime.js'
    at Object.fs.openSync (fs.js:646:18)
    at Object.fs.readFileSync (fs.js:551:33)
    at Engine.execute (/Users/matt/.nvm/versions/node/v8.11.1/lib/node_modules/@accordproject/cicero-cli/node_modules/@accordproject/cicero-engine/lib/engine.js:249:31)
    at Template.fromDirectory.then (/Users/matt/.nvm/versions/node/v8.11.1/lib/node_modules/@accordproject/cicero-cli/lib/commands.js:123:31)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
    at Function.Module.runMain (module.js:695:11)
    at startup (bootstrap_node.js:188:16)
    at bootstrap_node.js:609:3
09:24:17 - info: 
```

**Comment by jeromesimeon** [comment]
I'm not sure how to replace this.  You may want to try again with Cicero after PR #52 has been merged.

**Comment by mttrbrts** [comment]
Related to this line https://github.com/accordproject/cicero/blob/294ce8e45ae7918ff3a6764e59c45b3ac1a6153a/packages/cicero-engine/lib/engine.js#L249

**Comment by mttrbrts** [comment]
Fixed in #55
